Mathlib/Algebra/Parity.lean

```diff
@@ -351,6 +352,14 @@ theorem odd_two_mul_add_one (m : α) : Odd (2 * m + 1) :=
   ⟨m, rfl⟩
 #align odd_two_mul_add_one odd_two_mul_add_one
 
+@[simp] lemma odd_add_self_one : Odd (m + m + 1) := ⟨m, by rw [two_mul]⟩
```

Comment:
This one should follow a more reasonable pair of lemmas that `Even (m + m)` and `Odd (e +1)` where `Even e`

Reply:
OK I've added `Even.add_one` and `Even.one_add` instead of the two less general versions they imply. This is sufficient as `even_add_self` already exists.